spinegeneric/cli/generate_figure.py

```diff
@@ -170,10 +170,10 @@
     'csa_t2': 'Cord CSA from T2w [$mm^2$]',
     'csa_gm': 'Gray Matter CSA [$mm^2$]',
     'dti_fa': 'Fractional anisotropy',
-    'dti_md': 'Mean diffusivity [$mm^2.s^-1$]',
-    'dti_rd': 'Radial diffusivity [$mm^2.s^-1$]',
+    'dti_md': 'Mean diffusivity [$mm^2.s^{-1}$]',
+    'dti_rd': 'Radial diffusivity [$mm^2.s^{-1}$]',
     'mtr': 'Magnetization transfer ratio [%]',
-    'mtsat': 'Magnetization transfer saturation [a.u.]',
+    'mtsat': 'Magnetization transfer saturation',
```

Comment:
i think it is actually a percentage (correct me if i'm wrong)
```suggestion
    'mtsat': 'Magnetization transfer saturation [%]',
```

Reply:
You are right 👍 
In the `compute_mtsat` function [docstring](https://github.com/neuropoly/spinalcordtoolbox/blob/3da4b2697093d45969691fd3e48d1a2f85ddc197/spinalcordtoolbox/qmri/mt.py#L34 ) is written: "Compute MTsat (in percent) and T1 map (in s) based on FLASH scans"
and percentage are also stated in original Helms et el., 2008 paper:

<img width="635" alt="image" src="https://user-images.githubusercontent.com/39456460/89318926-bf7c3380-d67f-11ea-8213-950f63f1b836.png">